examples/flutter_gallery/lib/demo/buttons_demo.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
```

Comment:
And, we're exploring Gallery design ideas specifically for iOS-flavor of gallery. But that's not an immediate thing.